### ISSUE TITLE
FicWad

### ISSUE THREAD

**Issue by bkromhout** [bkromhout/FictionDL#1]
There are 3 types of links we'll need to handle:
- Oneshots: http://ficwad.com/story/141696
- Chapters: http://ficwad.com/story/227776
- Stories: http://ficwad.com/story/227774

Notice that all of these links look exactly the same, so we'll have to parse the HTML to figure out what's what. Also, if we get a chapter page, we'll have to find the story index page using the select box.

NC-17 stories are sorta special here. If a oneshot is NC-17, it doesn't matter since we can still download it as we already have the link. However, if any of the chapters of a story are NC-17 (like this one), we won't be able to get a link to them from the story index page, and we can't really reasonably assume that there will be some single chapter that isn't rated NC-17 from which we can get in and get to the select box.

TL;DR, for stories which have at least one chapter rated NC-17, we'll have to log in to FicWad.
